### PR TITLE
add signup page layout 

### LINKED ISSUE #5: Create Signup page layout
Create the layout for Signup Page as we discussed earlier.
> Effected file: `/src/pages/authentication/Signup.tsx`.
> Preview url: `/signup`

**Only update the markup. No need to implement the logic.**

### PR DESCRIPTION
**Pull request by thatsuman**
This PR solves #5
- destop mode:
image

### CODE REVIEW

File: src/pages/authentication/Signup.tsx [hunk 1]
```diff
@@ -1,3 +1,125 @@
...
+                  value=''
+                  className='w-4 h-4 border border-gray-300 rounded bg-gray-50 focus:ring-3 focus:ring-blue-300'
+                />
+              </div>
+
+              {/* for final release */}
+              {/* <label className="ml-2 text-sm font-medium text-gray-900">I agree with the <Link to="#" className="text-blue-600 hover:underline">terms and conditions</Link> and <Link to="#" className="text-blue-600 hover:underline">privacy policy</Link></label> */}
+
+              <label className='ml-2 text-sm font-medium text-gray-900'>
+                I agree to pay the said amount whenever Opencampus claims
```
Comment (asimbera) [inline src/pages/authentication/Signup.tsx:89]: Don't. Just don't.

File: src/pages/authentication/Signup.tsx [hunk 2]
```diff
@@ -1,3 +1,125 @@
...
+                  type='password'
+                  id='password'
+                  className='w-full p-4 pr-12 text-sm border-gray-200 rounded-lg shadow-sm'
+                  placeholder='Enter password'
+                />
+              </div>
+            </div>
+
+            <div>
+              <label className='text-sm font-medium'>Confirm Password</label>
```
Comment (asimbera) [inline src/pages/authentication/Signup.tsx]: No need for confirm password field. Will add visibility toggle for password field later on.

**Review by asimbera — CHANGES_REQUESTED** [review]

File: src/pages/authentication/Signup.tsx (hunk 1 above)
Comment (thatsuman) [reply src/pages/authentication/Signup.tsx:89]: Please allow them to stay 😢  let us do some business. Also added a fresh checkbox for later use.

File: src/pages/authentication/Signup.tsx (hunk 2 above)
Comment (thatsuman) [reply src/pages/authentication/Signup.tsx]: Ok, I'll remove it.

File: src/pages/authentication/Signup.tsx [hunk 3]
```diff
@@ -1,3 +1,113 @@
...
+            <div className='relative mt-1'>
+              <label className='text-sm font-medium'>First Name</label>
+              <div className='relative mt-1'>
+                <input
+                  type='text'
+                  id='firstname'
+                  className='w-full p-4 pr-12 text-sm border-gray-200 rounded-lg shadow-sm'
+                  placeholder='Enter First Name'
+                />
+              </div>
```
Comment (asimbera) [inline src/pages/authentication/Signup.tsx]: Add for attribute in label tag referencing the id of the input field.


```suggestion
              <label className='text-sm font-medium' for='firstname'>First Name</label>
              <div className='relative mt-1'>
                <input
                  type='text'
                  id='firstname'
                  className='w-full p-4 pr-12 text-sm border-gray-200 rounded-lg shadow-sm'
                  placeholder='Enter First Name'
                />
              </div>
```

Do this for all labels.

**Review by asimbera — CHANGES_REQUESTED** [review]

**Review by asimbera — APPROVED** [review]

**Pull request merged**